### ISSUE TITLE
I get repeated authentication failure

### ISSUE THREAD

**Issue by jeffslofish** [vccw-team/vccw#127]
I followed all the installation instructions, but when it gets to the part in the script below, it hangs on an authentication failure. I get the same results on OSX 10.11 and Windows 10. There is a thread about this sort of issue [here](https://github.com/mitchellh/vagrant/issues/5186), but there is no clear resolution.

    vccw.dev: SSH address: 127.0.0.1:2200
    vccw.dev: SSH username: vagrant
    vccw.dev: SSH auth method: private key
    vccw.dev: Warning: Remote connection disconnect. Retrying...
    vccw.dev: 
    vccw.dev: Vagrant insecure key detected. Vagrant will automatically replace
    vccw.dev: this with a newly generated keypair for better security.
    vccw.dev: 
    vccw.dev: Inserting generated public key within guest...
    vccw.dev: Removing insecure key from the guest if it's present...
    vccw.dev: Key inserted! Disconnecting and reconnecting using new SSH key...
    vccw.dev: Warning: Authentication failure. Retrying...
    vccw.dev: Warning: Authentication failure. Retrying...
    vccw.dev: Warning: Authentication failure. Retrying...
    vccw.dev: Warning: Authentication failure. Retrying...
    vccw.dev: Warning: Authentication failure. Retrying...
    vccw.dev: Warning: Authentication failure. Retrying...
    vccw.dev: Warning: Authentication failure. Retrying...
    vccw.dev: Warning: Authentication failure. Retrying...

**Comment by raulillana** [comment]
Just keep waiting. Hangs for a while but then it suddenly works.

Anyway would be nice to get this fixed… :_)

**Comment by jeffslofish** [comment]
I waited about 4 minutes, then I got this message. It doesn't continue after this.

    vccw.dev: Warning: Authentication failure. Retrying...
    vccw.dev: Warning: Authentication failure. Retrying...
    vccw.dev: Warning: Authentication failure. Retrying...
    vccw.dev: Warning: Authentication failure. Retrying...
    vccw.dev: Warning: Authentication failure. Retrying...
    vccw.dev: Warning: Authentication failure. Retrying...
    vccw.dev: Warning: Authentication failure. Retrying...
    vccw.dev: Warning: Authentication failure. Retrying...
    vccw.dev: Warning: Authentication failure. Retrying...
    vccw.dev: Warning: Authentication failure. Retrying...
Timed out while waiting for the machine to boot. This means that
Vagrant was unable to communicate with the guest machine within
the configured ("config.vm.boot_timeout" value) time period.

If you look above, you should be able to see the error(s) that
Vagrant had when attempting to connect to the machine. These errors
are usually good hints as to what may be wrong.

If you're using a custom box, make sure that networking is properly
working and you're able to connect to the machine. It is a common
problem that networking isn't setup properly in these boxes.
Verify that authentication configurations are also setup properly,
as well.

**Comment by torounit** [comment]
Do you use a vagrant 1.8.5 ?

The issue is vagrant bug.
https://github.com/mitchellh/vagrant/issues/7610

Maybe it should be solved by using 1.8.4.

**Comment by jeffslofish** [comment]
Yes, I was using Vagrant 1.8.5 with VirtualBox 5.1.4. I was able to get it working with Vagrant 1.8.4, but I had to first downgrade my VirtualBox to 5.0.26 because Vagrant was complaining about not finding a provider.

Also, I did try building Vagrant from sources and was eventually successful at producing a Vagrant binary (1.8.6.dev) that seemed to do the trick as well.

A third way to get around the problem I tried, which I found  [here](https://github.com/mitchellh/vagrant/issues/5186) was to set 

```
config.ssh.username = "vagrant" 
config.ssh.password = "vagrant"
```

in the Vagrant file.

So ultimately, this is supposed be resolved in the next release of Vagrant, but in the meantime there are a few workarounds!
